### PR TITLE
Add password policy api postman collection

### PR DESCRIPTION
**Pull request by dannavarro-okta**
Resolves: OKTA-112100
Bacon:test

- This collection should stay private until otherwise noted.
- We will make the collection public when GPP API is EAed...
- ...Or it will be bundled into a folder in the Policies API collection, when that is GAed

@mystiberry-okta @edjohns-okta

### CODE REVIEW

**Comment by johngronberg-okta** [comment]
@mystiberry-okta do we need to add this to our postman cloud account as well?

**Comment by mystiberry-okta** [comment]
@johngronberg-okta Yes, but not yet. It's still Beta, right @dannavarro-okta ?

File: _source/_docs/api/postman/password-policy.json [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    }
+  ]
+}
```
Comment (mystiberry-okta) [inline _source/_docs/api/postman/password-policy.json]: Do we need a newline here or is github confused?

**Comment by dannavarro-okta** [comment]
@johngronberg-okta @mystiberry-okta we are keeping this private for now. Holding off until we EA the API, which should be in roughly 1 month

File: _source/_docs/api/postman/password-policy.json (hunk 1 above)
Comment (dannavarro-okta) [inline _source/_docs/api/postman/password-policy.json]: I think that github is just confused. none of the other collections have a new line at the end either

**Pull request merged**